example/foo.go

```diff
@@ -31,7 +31,7 @@ type FooResponse struct {
 	Instruction   Instruction            `json:"instruction"`
 	BsonPtr       *BsonID                `json:"bsonPtr,omitempty" example:"blah blah blah"`
 	RandomBool    bool                   `json:"randomBool,omitempty" example:"true"`
-	MyEnum        string                 `json:"myEnum" goas:"enum=value1 value2"`
+	MyEnum        string                 `json:"myEnum" goas:"enum=value1 value2,required=true"`
```

Comment:
Based on the implementation in `parser.go`, shouldn't this just be `required`, not `required=true`?